server/controllers/follow.controllers.js

```diff
@@ -94,7 +100,9 @@ const followController = {
       // Confirm if user is already unfollowing
       if (!followRecord) {
         return res.status(404).json({
-          message: 'You are already unfollowing user',
+          errors: {
+            body: ['You are already unfollowing user'],
```

Comment:
This can be something like - `You are not currently following this user`

Reply:
Noted and fixed